### ISSUE TITLE
[BUG] AOIassist script

### ISSUE THREAD

**Issue by dbekaert** [aria-tools/ARIA-tools#172]
**Describe the bug**
Error for running AOIassist script.

**To Reproduce**
1. Download 
```
https://api.daac.asf.alaska.edu/services/search/param?intersectsWith=POLYGON((-96.5828%2064.1841,-80.2435%2064.1841,-80.2435%2071.1638,-96.5828%2071.1638,-96.5828%2064.1841))&platform=SENTINEL-1&relativeOrbit=77&processinglevel=SLC&beamSwath=IW&maxResults=5000&output=CSV
```

2. Command used
```
ariaAOIassist.py -f Boothia_Peninsula\ .csv
```
3. Error Output
```
IndexError: list index out of range
(ARIA-tools_latest) MT-101939:NISAR_ST dbekaert$ ariaAOIassist.py -f Boothia_Peninsula\ .csv 
Generating outputs for track: 77
Traceback (most recent call last):
  File "/Users/dbekaert/Software/python/miniconda37/envs/ARIA-tools_latest/bin/ariaAOIassist.py", line 720, in <module>
    track_metadata.plotFrameCenters()
  File "/Users/dbekaert/Software/python/miniconda37/envs/ARIA-tools_latest/bin/ariaAOIassist.py", line 404, in plotFrameCenters
    self.ax.set_xlim([dates[0]-timedelta(days=6),dates[-1]+timedelta(days=6)])
```

**Comment by dbekaert** [comment]
@rzinke running in some other errors when enabling the approximate AOI option.
AttributeError: 'GeometryCollection' object has no attribute 'exterior'

**Comment by rzinke** [comment]
Hi @dbekaert, I tracked down the source of the problem. It is not a bug, but stems from the fact that not all the acquisition epochs cover the same area. Notice how the first two dates of acquisition do not share a common extent, therefore, the program does not know what AOI you want to construct.

I made some modifications so that the routines now automatically exclude epochs for which the spatial extent is not complete. I also added a few checks, warnings, and error messages for cases in which things get wonky. I'll push the changes now, and if they look good I'll update the ARIA-tools-docs.
